lib/image.ex

```diff
@@ -1276,7 +1276,7 @@ defmodule Image do
   * `{:error, reason}`
 
   """
-  @spec resize(Vimage.t(), width :: pos_integer(), options :: Resize.resize_options()) ::
+  @spec resize(Vimage.t(), width :: pos_integer() | binary(), options :: Resize.resize_options()) ::
```

Comment:
I think this change, to add `| binary()` shouldn't be necessary because you've fixed the type for dimensions on below. I will need to do some tests because dialyzer doesn't always combine specs in the way I would expect.